src/voice-pe/customizing/grove-port.md

```diff
@@ -16,6 +16,11 @@ Connecting a sensor to the Grove port of the device allows the sensor to use the
 
 ## Prerequisites
 
+{% steps %}
+{% step " " " " %}
```

Comment:
This is a workaround that will be prone to problems in the future, which will go undetected.
As an alternative to this there should be a `intoStep` (or similar) without these arguments applied at all.

Reply:
> This is a workaround that will be prone to problems in the future, which will go undetected. As an alternative to this there should be a `intoStep` (or similar) without these arguments applied at all.

https://github.com/NabuCasa/support/pull/518

Reply:
Like this? https://github.com/NabuCasa/support/pull/518

Reply:
yes